app/ui/src/app/integration/edit-page/flow-view/flow-view.component.scss

```diff
@@ -110,32 +123,40 @@
       width: 2px;
       background-color: #d4d4d4;
     }
+
     &.finish {
       flex: 0 0 auto;
       min-height: 0px;
     }
+
     &.step-insert {
       flex: 0 0 auto;
       min-height: 80px;
+
       /deep/ .popover {
```

Comment:
Now that you're editing this file, would you mind replacing this `/deep/` Shadow Piercing combinator by the LTS one (`::ng-deep`)? Thanks.

Whenever you see something that you can improve while you're introducing some other changes in that file, do not hesitate and proceed. The idea is to leave the code in a better state that we found it.

Reply:
Done, also now importing syndesis-sass instead of patternfly-sass.